site/src/components/Dashboard/useUpdateCheck.ts

```diff
@@ -0,0 +1,49 @@
+import { updateCheck } from "api/queries/updateCheck";
+import { useMemo, useState } from "react";
+import { useQuery } from "react-query";
+
+type UpdateCheckState = "show" | "hide";
```

Comment:
Do you think there would be more states added down the line? Right now, I'm wondering if this could be represented as something like `visible: true | false`

Reply:
I don't think so, we can simplify that for sure.